### ISSUE TITLE
Is there a way to start it on the daemon level?

### ISSUE THREAD

**Issue by RyanOrigens** [anufrievroman/waypaper#17]
It is very annoying for me, everytime i start my computer it takes a couple seconds before it changes the wallpaper...

**Comment by anufrievroman** [comment]
This is basically a program to interact with deamons, it's not a deamon itself, so performance issues are usually related to the backends. If you added `waypaper --restore` to your startup script, it should just start the deamon of your chosen backend instantly. So, I suspect there's something with your startup or the chosen backend... A couple of questions:
1) which backend you are using and on what kind of system?
2) if in your startup script you use a command for your backend directly instead of `waypaper --restore`, does it work instantly?

**Comment by RyanOrigens** [comment]
I'm using swaybg and hyprland.

This is not only waypaper issue, this is hyprland thing, it takes a while since it need to wait hyprland load.

It also happened when i was using hyprpaper istead.

by deamon i mean i wanted to start it as a service, so it would start on my display manager rather than when i start hyprland.

**Comment by anufrievroman** [comment]
I use the same setup actually and for me hyperland and wallpaper load very quickly even on a super old laptop, so I still suspect something might be off in your setup, maybe order of things to load first?
I am not sure even if you can preload wallpaper deamon before loading into actual WM ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

**Comment by RyanOrigens** [comment]
It loads fast, the problem is hyprland shows another image before showing it, it takes around 1/2 seconds but there is a annoying anime image that irritates me in there.

It is in my first line in hyprland, i wanted to never see that image.

**Comment by anufrievroman** [comment]
But where is that initial image is coming from? I mean, something must set that first wallpaper?

**Comment by RyanOrigens** [comment]
Yes, it is the default on hyprland, it comes with a few images. I'll be closing the issue since this is probably an issue with hyprland setting a default wallpaper rather than an waypaper issue.
